sql/command_mapping.cc

```diff
@@ -206,7 +206,16 @@ const char *Command_maps::sql_commands[] = {"select",
                                             "unlock_instance",
                                             "restart_server",
                                             "create_srs",
-                                            "drop_srs"};
+                                            "drop_srs",
+                                            "show_parse_tree",
```

Comment:
According to documentation `SHOW PARSE_TREE` is a debug-only upstream statement
https://dev.mysql.com/doc/refman/8.4/en/show-parse-tree.html
However, I do not see any conditional compilation around `SQLCOM_SHOW_PARSE_TREE`.
Just wondering if this changes anything.